db/migrate/20200720092916_add_cron_to_delayed_jobs.rb

```diff
@@ -0,0 +1,7 @@
+class AddCronToDelayedJobs < ActiveRecord::Migration[6.0]
+  def change
+    add_column :delayed_jobs, :cron, :string
+
+    Delayed::Job.enqueue(TranscodeCacheCleanJob.new, cron: '0 0 * * *')
```

Comment:
Assuming the server is in a similar timezone to the user, I would shedule this a few hours later to get as little overlap with actual usage.

Reply:
Sure, I'll edit. Overlap with usage shouldn't matter though.